src/battle_main.c

```diff
@@ -4412,6 +4412,10 @@ s8 GetMovePriority(u32 battlerId, u16 move)
             break;
         }
     }
+    if (gProtectStructs[battlerId].quash == 1)
```

Comment:
to match pret styling, can you add a space between lines 4414 and 4415? also, remove the braces. Also remember spaces instead of tabs